shared/src/main/java/uk/gov/di/authentication/shared/services/AuthenticationAttemptsService.java

```diff
@@ -40,9 +40,9 @@ public void addCode(
     }
 
     public void createOrIncrementCount(
-            String internalSubjectId, long ttl, String authenticationMethod, String journeyType) {
+            String internalSubjectId, long ttl, String journeyType, String countType) {
```

Comment:
Including the ttll in the service would imply having the configuration service included in the service as well. This would need changes to the base handler, which is inherited by 9 classes across the service. Moreover, making the configuration service static is impossible as the classes are shared with orch.